dashboards-observability/public/components/visualizations/charts/bar/bar_type.ts

```diff
@@ -17,19 +17,19 @@ import {
   ConfigLegend,
   InputFieldItem,
 } from '../../../event_analytics/explorer/visualizations/config_panel/config_panes/config_controls';
-import { DefaultChartStyles } from '../../../../../common/constants/shared';
+import { DefaultChartStyles, visChartTypes } from '../../../../../common/constants/shared';
 
 import { ConfigColorTheme } from '../../../event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_color_theme';
 const sharedConfigs = getPlotlySharedConfigs();
 const VIS_CATEGORY = getPlotlyCategory();
 
 const { LegendPosition, ShowLegend } = DefaultChartStyles;
 export const createBarTypeDefinition = (params: any) => ({
-  name: 'bar',
+  name: params.type ? params.type : 'bar',
   type: 'bar',
-  id: 'bar',
-  label: 'Bar',
-  fullLabel: 'Bar',
+  id: params.type ? params.type : 'bar',
+  label: params.type === visChartTypes.HorizontalBar ? 'Horizontal Bar' : 'Bar',
```

Comment:
Don't we need to change the labels as `Vertical Bar`? What would be the difference between `Bar` and `Horizontal Bar`? `Horizontal Bar` is also a `Bar` anyway.

Reply:
Yes, we have renamed Bar to Vertical Bar in next PR.